Dockerfile-alpine.template

```diff
@@ -0,0 +1,69 @@
+FROM openjdk:8-alpine
+
+ENV ROOTDIR /usr/local/
+ENV LD_LIBRARY_PATH /usr/local/lib
+ENV GDAL_VERSION %%GDAL_VERSION%%
+ENV OPENJPEG_VERSION %%OPENJPEG_VERSION%%
+
+ENV ANT_VERSION %%ANT_VERSION%%
+ENV ANT_HOME $ROOTDIR/src/apache-ant-${ANT_VERSION}/
+ENV PATH ${ANT_HOME}/bin:${PATH}
+
+# Load assets
+WORKDIR $ROOTDIR/
+
+ADD http://download.osgeo.org/gdal/${GDAL_VERSION}/gdal-${GDAL_VERSION}.tar.gz $ROOTDIR/src/
+ADD https://github.com/uclouvain/openjpeg/archive/v${OPENJPEG_VERSION}.tar.gz $ROOTDIR/src/openjpeg-${OPENJPEG_VERSION}.tar.gz
+ADD https://www.apache.org/dist/ant/binaries/apache-ant-${ANT_VERSION}-bin.tar.gz $ROOTDIR/src/ant-${ANT_VERSION}.tar.gz
+
+# Install basic dependencies
+RUN apk update && apk add \
```

Comment:
I think that this use of `apk` should be modified to remove build dependencies after we're done with them. Unfortunately, that might require sifting through these to determine which are true build dependencies vs. runtime ones.

https://github.com/azavea/docker-flask/blob/f9433b9a4d62336d9914b386ef9b178f93630c92/Dockerfile-alpine.template#L7-L12

(Note how the `pip install` step that needs to build dependencies is wrapped by the install and uninstall steps.)